web/src/components/Navigation/TopNav.tsx

```diff
@@ -116,42 +123,44 @@ export default function TopNav({ searchValue, setSearchValue }: Props) {
                 className={"w-11/12 max-w-xl sm:w-2/3"}
               />
               <LinkIconButton
-                href={"/create"}
+                href={`/create?prompt=${searchValue}`}
                 className={"hidden text-base sm:block"}
               >
                 <MdAddCircle className={"h-full w-10"} />
               </LinkIconButton>
             </motion.li>
           )}
 
-          {isProfile ? (
+          {(!isProfile || !isOwnProfile) && (
```

Comment:
I think it's safe to reduce this do `!isOwnProfile` since that variable is already dependent on `isProfile`.

Reply:
Then you can actually make this into a ternary, combined with the below